release/kata-deploy-binaries.sh

```diff
@@ -109,15 +109,7 @@ install_image() {
 
 #Install kernel asset
 install_kernel() {
-	if [[ "$test_local" == "true" ]]; then
-		pushd "${script_dir}/../"
-	else
-		go get "github.com/${project}/packaging" || true
-		pushd ${GOPATH}/src/github.com/${project}/packaging >>/dev/null
-		git checkout "${kata_version}-kernel-config" ||
-		git checkout "${kata_version}"
-	fi
-
+	pushd "${script_dir}/../"
```

Comment:
I had errros in the past, the kernel build failed to build with a newer version, I know this wont affect with the new actions model, but  could you explain/comment  what is the error ?